### ISSUE TITLE
Special projects allocation between boats

### ISSUE THREAD

**Issue by RebeccaHoward-NOAA** [afsc-gap-products/GOA-AI-Survey-Planning#187]

**Comment by sean-rohan-NOAA** [comment]
Is this related to station allocation? If so, Zack already been produced the file and it's available on G:\RACE_Charts and the navmaps repo.

**Comment by RebeccaHoward-NOAA** [comment]
I believe this has to do with which special projects are assigned to each vessel, @bethanyriggle is that correct?

**Comment by bethanyriggle** [comment]
@RebeccaHoward-NOAA this is correct. The label says "special projects" thus, it is the allocation of special projects between boats.

**Comment by AlexandraDowlin-NOAA** [comment]
I wanted to add some information here for Sean and Bianca that are on this task. This in regards to assigning what projects will be deployed on either vessel. Once we get past what projects are accepted, we will separate projects based on vessel capacity (i.e. the AKP normally has most of the large frozen fish projects since they have a larger freezer capacity).

**Comment by sean-rohan-NOAA** [comment]
Thank you, all. Appreciate the clarification.

**Comment by AlexandraDowlin-NOAA** [comment]
Bianca, Sean Rooney and I met yesterday, 3/11, to talk about vessel allocation of scientific collections. We were able to assign projects to the two vessels but wanted to loop in @Ned-Laman-NOAA and @smcdermo that this initial allocation may change over the next few weeks depending on survey dates/crew changes/freezer capacity on AKP.

**Comment by Ned-Laman-NOAA** [comment]
So noted. Post here when you know the status of the AKP hockey rink-sized freezer.
